src/App.css

```diff
@@ -25,6 +25,11 @@
 
 .facet ul {
   text-align: left;
+  padding-left: 10px;
```

Comment:
Actually I think I would rather not modify this CSS. In this case as well, I think it's better to keep it very minimal so learners understand that Headless only takes care of the application state, and that users have to handle the CSS themselves.